import-scripts/generate_cdd_attribute_tsv.py

```diff
@@ -63,6 +62,8 @@
 import re
 import requests
 import sys
+from unidecode import unidecode
+#this program requires "pip install unidecode"
```

Comment:
let's move this comment to the commit message

Reply:
also just fyi I have now installed this module for our python3 installation for the cbioportalimporter user